src/plugins/hiddenRows/test/selection.e2e.js

```diff
@@ -0,0 +1,1429 @@
+describe('HiddenRows', () => {
```

Comment:
IMO same situation as with `getCoords`. Selection should be tested in core with `indexMapper.register(HiddenMap)` and those tests should pass whenever hidden rows are a result of hiddenRows plugin or any other

Reply:
Same with fixedRowsBottom.e2e, fixedRowsTop.e2e, rowHeaders.e2e

Reply:
Moved the task to the separate issue https://github.com/handsontable/handsontable/issues/6908.